java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbDataWriter.java

```diff
@@ -127,7 +127,13 @@ private void addRow(InternalRow record, PreparedStatement preparedStatement) thr
     @Override
     public WriterCommitMessage commit() throws IOException {
         try {
-            preparedStatement.executeBatch();
+            int[] rc = preparedStatement.executeBatch();
+            for(int i = 0; i < rc.length; i++) {
+                int code = rc[i];
+                if (code < 0) {
+                    throw new SQLException("List of " + i + " insert failed. Response code: " + code);
```

Comment:
About the execption message, `i` is useless, we can't find which row is the i th row. And `rc[i]` is just a int, not a code.
Count how many elem in rc is -1, report `insert X rows, Y rows failed`.

Reply:
ok